File: Lab5java/app/src/main/java/pe/edu/upc/lab5java/MainActivity.java
package pe.edu.upc.lab5java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void btnGet(View view) {
        startActivity(new Intent(MainActivity.this, ProductosBuscarActivity.class));
    }

    public void btnPost(View view) {
        startActivity(new Intent(MainActivity.this, ProductoRegistrarActivity.class));
    }
}

File: Lab4Java/app/src/main/java/pe/edu/upc/lab4java/MainActivity.kt
package pe.edu.upc.lab4java

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun btnInsertar(view: View) {
        Toast.makeText(this@MainActivity, "Click on Insertar", Toast.LENGTH_SHORT).show()
        Log.i("#####", "Click on Insertar!")
        startActivity(Intent(this, GeneroMusicalNuevoActivity::class.java))
    }

    fun btnBuscar(view: View) {
        Toast.makeText(this@MainActivity, "Click on Buscar", Toast.LENGTH_SHORT).show()
        Log.i("#####", "Click on Buscar!")
        startActivity(Intent(this, GeneroMusicalBuscarActivity::class.java))
    }

    fun btnEscribirSP(view: View) {
        Toast.makeText(this@MainActivity, "Click on EscribirSP", Toast.LENGTH_SHORT).show()
        Log.i("#####", "Click on EscribirSP!")
        startActivity(Intent(this, EscribirPreferenciaActivity::class.java))
    }

    fun btnLeerSP(view: View) {
        Toast.makeText(this@MainActivity, "Click on LeerSP", Toast.LENGTH_SHORT).show()
        Log.i("#####", "Click on LeerSP!")
        startActivity(Intent(this, LeerPreferenciaActivity::class.java))
    }
}